src/pages/Product/ProductDescription/ProductDescription.tsx

```diff
@@ -0,0 +1,33 @@
+import { ReactElement, memo } from 'react';
+import { useSelector } from 'react-redux';
+import { splitToParagraphs } from '../../../utils';
+import { Accordion } from '../../../components/shared/Accordion';
+import { productAccordionData } from '../../../constant';
+import { RootState } from '../../../store/store';
+
+import styles from './productDescription.module.scss';
+
+function ProductDescriptionComp(): ReactElement {
+  const { product } = useSelector((store: RootState) => store.product);
```

Comment:
Maybe, it will be better to add selector to store, if you use this data in several components?
Instead of '(store: RootState) => store.product' will be some selector.

Reply:
Done.